frontend/src/metabase/css/components/icons.css

```diff
@@ -11,28 +11,6 @@
   height: 42.5px;
 }
 
-@keyframes icon-pulse {
-  0% {
-    box-shadow: 0 0 5px var(--color-shadow);
-  }
-  50% {
-    box-shadow: 0 0 5px var(--color-shadow);
-  }
-  100% {
-    box-shadow: 0 0 5px var(--color-shadow);
-  }
-}
-
-.Icon--pulse {
-  border-radius: 99px;
-  box-shadow: 0 0 5px var(--color-shadow);
-  padding: 0.75em;
-  animation-name: icon-pulse;
-  animation-duration: 2s;
-  animation-iteration-count: infinite;
-  animation-timing-function: linear;
-}
```

Comment:
Although we do have `.Icon-pulse` in `frontend/src/metabase/icon_paths.js`, I believe this is something different (note the difference between a single and a double dash). Probably some old version of a spinner?

In any case:
1. I didn't find it used anywhere in the project
2. Looking at its keyframes, it doesn't do anything